module/zfs/vdev_raidz.c

```diff
@@ -3624,7 +3611,7 @@ vdev_raidz_get_tsd(spa_t *spa, nvlist_t *nv)
 		 */
 	}
 
-	avl_create(&vdrz->vre_txgs, vedv_raidz_reflow_compare,
+	avl_create(&vdrz->vd_expand_txgs, vedv_raidz_reflow_compare,
 	    sizeof (reflow_node_t), offsetof(reflow_node_t, re_link));
 
 	error = nvlist_lookup_uint64_array(nv, ZPOOL_CONFIG_RAIDZ_EXPAND_TXGS,
```

Comment:
Seems like, there is an issue in case of export/import during first expansion, when tree is empty.
Possible the next check could be added:
+       } else {
+               if (vdrz->vn_vre.vre_offset != UINT64_MAX)
+                       vdrz->vd_original_width-=1;
        }

Reply:
Good point.  I think that problem would occur when importing during any expansion, not just the first.